teku/src/main/java/tech/pegasys/teku/cli/options/MetricsOptions.java

```diff
@@ -119,6 +119,18 @@ public class MetricsOptions {
   private boolean blockProductionPerformanceEnabled =
       MetricsConfig.DEFAULT_BLOCK_PRODUCTION_PERFORMANCE_ENABLED;
 
+  @Option(
+      names = {"--Xmetrics-block-production-timing-tracking-warning-threshold"},
+      hidden = true,
+      showDefaultValue = Visibility.ALWAYS,
+      paramLabel = "<INTEGER>",
+      description =
+          "Whether block production is considered slow if it takes longer than this threshold (in ms)",
```

Comment:
I think description could be improved but let's pass it to english-speaker

Reply:
The author is copilot :)

Reply:
do you like the new version?

Reply:
not really...
```
The time (in ms) at which block production is to be considered 'slow'. If set to 100, block production taking at least 100ms would raise a warning.
```

Reply:
updated